content/03.results.md

```diff
@@ -194,18 +198,9 @@ Additionally, we found a significant positive correlation between _TP53_ scores
 Tumors with the highest median _TP53_ scores were those known to harbor somatic _TP53_ alterations: choroid plexus tumors, embryonal tumors, HGATs, and pineal tumors (**Figure {@fig:Fig5}F**), while melanocytic tumors, meningiomas, and tumors of cranial and paraspinal nerves had the lowest scores.
 
 We next used gene expression data to predict telomerase activity using EXpression-based Telomerase ENzymatic activity Detection (EXTEND) [@doi:10.1038/s41467-020-20474-9] as a surrogate measure of malignant potential [@doi:10.1038/s41467-020-20474-9; 10.1093/carcin/bgp268].
-EXTEND scores significantly correlated with _TERT_ and _TERC_ expression (**Figure {@fig:FigS5}C-D**).  
+EXTEND scores significantly correlated with _TERT_ and _TERC_ expression (**Figure {@fig:FigS5}G-H**).  
 We found aggressive tumors such as CNS lymphoma, ETMR, ATRT, DMG, and HGG had high EXTEND scores (**Figure {@fig:Fig5}G**), while benign lesions such as teratomas, dysplasias, and hemangioblastomas had the lowest scores (**Table S3**).
 
 <!-- TODO: Add link to S3 once generated. -->
 
-![**Figure 5. _TP53_ and telomerase activity** A, Receiver Operating Characteristic for _TP53_ classifier run on FPKM of stranded RNA-Seq samples.
-B, Violin and box plots of _TP53_ scores plotted by _TP53_ alteration type.
-C, Violin and box plots of _TP53_ RNA expression plotted by _TP53_ activation status.
-D, Violin and box plots of _TP53_ scores in Li Fraumeni patient tumors.
-E, _TP53_ scores significantly correlate with CNV breakpoint density across PBTA tumors (Pearson R = 0.25, p = 7.5e-12).
-F, Box plots of _TP53_ scores grouped by `broad_histology`.
-G, Box plots of EXTEND scores grouped by `broad_histology`. 
-H, _TP53_ scores significantly correlate with EXTEND scores across PBTA tumors (Pearson R = 0.31, p <2.2e-16).
-I, Histology-specific Pearson correlations between _TP53_ and EXTEND scores.
-J, Kaplan-Meier](images/main/Figure5.png){#fig:Fig5}
+![**Figure 5. _TP53_ and telomerase activity** A, Receiver Operating Characteristic for _TP53_ classifier run on FPKM of stranded RNA-Seq samples. B, Violin and box plots of _TP53_ scores plotted by _TP53_ alteration type. C, Violin and box plots of _TP53_ RNA expression plotted by _TP53_ activation status. D, Violin and box plots of _TP53_ scores in Li Fraumeni patient tumors. E, _TP53_ scores significantly correlate with CNV breakpoint density across PBTA tumors (Pearson R = 0.25, p = 7.5e-12). F, Box plots of _TP53_ scores grouped by `broad_histology`. G, Box plots of EXTEND scores grouped by `broad_histology`. H, _TP53_ scores significantly correlate with EXTEND scores across PBTA tumors (Pearson R = 0.31, p <2.2e-16). I, Histology-specific Pearson correlations between _TP53_ and EXTEND scores. J, Kaplan-Meier](images/main/Figure5.png){#fig:Fig5}
```

Comment:
I think we need to hold off on this legend since some discussion still needs to happen for a final version of this figure. For example, `D` as is doesn't have a meaningful sample size to interpret so this panel is misleading. `H` and `I` are statistically confounded (one is subgroupings of another). As far as I know the `J` panel isn't actually meant to be part of the figure either(?).

Reply:
oh yes, I should have updated that here - did that in a different PR 🤦